style.css

```diff
@@ -0,0 +1,421 @@
+
+/* google font  */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+:root {
+  --clr-purple: #6070ff;
+  --clr-white: #fff;
+  --clr-para: #396df2;
+  --clr-gray-100: #808080;
+}
+
+/* Reset style for page */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+/* navbar styles */
+.header-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem;
+}
+
+.topNav {
+  display: none;
+}
+
+.nav-items {
+  margin: 0 0.5rem;
+}
+
+.nav-brand {
+  font-weight: 700;
+  color: #6070ff;
+}
+
+/* Hamburger menu */
+.hamburger {
+  display: block;
+}
+
+.bar {
+  display: block;
+  height: 5px;
+  width: 35px;
+  background-color: #6070ff;
+  margin: 3px;
+}
+
+/* background shadow  */
+.box2 {
+  display: flex;
+  flex-direction: column;
+  box-shadow: -10px 10px 10px -6px rgba(207, 205, 205, 0.501);
+  padding-bottom: 0;
+  border-bottom-left-radius: 3rem;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.Greetings {
+  margin: 0 0;
+}
+
+.Greetings h1 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 4rem;
+  line-height: 60px;
+  color: #091e42;
+  padding: 5rem 1rem 1rem 1rem;
+  padding-bottom: 1rem;
+}
+
+main {
+  display: grid;
+}
+
+.Content {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding: 1rem;
+}
+
+.contact {
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 24px;
+  color: #7f8cff;
+  padding: 1rem;
+}
+
+.social-icons {
+  display: flex;
+  gap: 1rem;
+  padding: 0 1rem;
+  align-items: center;
+  margin-top: 0.5rem;
+}
+
+.social-icons svg {
+  width: 1.5rem;
+  margin-top: 3rem;
+  margin-bottom: 3rem;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  background: var(--clr-white);
+  border: 1px solid #dfe1e6;
+  padding: 1rem;
+  gap: 12px;
+  border-radius: 1rem;
+  margin: 2rem 2rem 0 2rem;
+  box-shadow: 8px 8px 10px -10px rgb(209, 204, 204);
+}
```

Comment:
One important point about this project is to be aligned with the original design. Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉. For this reason, check below the differences between your work section compared to Figma's:
- Please increase the margin (black) and decrease the padding (blue):

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/205344880-d2e084a7-935e-4a21-a6e2-20e6879da8ab.png)  | ![image](https://user-images.githubusercontent.com/42799579/205344915-7833d5a2-37b0-42cc-b846-243601564ed2.png)  |

- Please make sure to use the correct color & font-weight for the texts:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/205345287-df040ac2-2c6c-46ef-8b65-75f709ee71b3.png)  | ![image](https://user-images.githubusercontent.com/42799579/205345265-4f1ff46f-05f8-41bf-b902-32ea63ca85f6.png)  |

- Kindly notice the dots in the original design:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/205345453-0f88ea45-3fe5-4026-8e50-22b30fa446b9.png)  | ![image](https://user-images.githubusercontent.com/42799579/205345407-7d5b6c1b-739d-4ec1-ae98-f95a856d6e0a.png)  |

By updating these points, your project will look more like the original, displaying your front-end abilities :smiley: